wiki/Development-Practices.md

```diff
@@ -0,0 +1,187 @@
+
+## License
+
+[Animate](https://github.com/mesh-adaptation/animate/blob/main/LICENSE), [Goalie](https://github.com/mesh-adaptation/goalie/blob/main/LICENSE), and [Movement](https://github.com/mesh-adaptation/movement/blob/main/LICENSE) have MIT licenses. Familiarise yourself with their terms.
+
+## Setting up development environment
+
+To prepare for the development of the Mesh Adaptation package you are interested in, please follow the [installation instructions](../Installation-Instructions#installing-mesh-adaptation-modules), but replace `make install` with `make install_dev`. This will install development dependencies (such as pytest for testing and Ruff for code linting and formatting), in addition to mandatory production dependencies. The full list of dependencies can be found in the `pyproject.toml` file of each package's repository.
+
+## GitHub comments
+
+* If you make a comment anywhere in the GitHub API (e.g., on an Issue or PR) and realise it wasn't quite right, **do not** delete the comment. Many users have GitHub configured to send them emails and trying to track down a deleted comment from an email can cause a lot of confusion.
+  * Instead of deleting the comment, edit it and use ~~strikeout~~ to delete the offending text. (To do this, put `~~` before and after the text you want to strike out.)
+  * For extra clarity, add another comment starting with "Edit:", which explains that what you said was incorrect and why.
+
+## Code comments
+
+* Make sure that comments are used wherever it isn't obvious what a piece of code is doing.
+* However, avoid adding comments when it is obvious, e.g.
+```
+# Rename the function
+f.rename("my_function")
+```
+
+## Projects
+
+### Mesh Adaptation development board
+
+The three Mesh Adaptation codes and their documentation repo have a shared [Project board](https://github.com/orgs/mesh-adaptation/projects/2/views/1) for ongoing development work.
+The project board categories are:
+* **Backlog** - Issues that have been flagged but there are not currently any plans to address them.
+* **Priority** - Issues that have been flagged as important and should be addressed as soon as possible.
+* **Blocked** - Issues that cannot be progressed until one or more other issues are addressed.
+* **In progress** - Issues for which one or more developers are currently working on addressing.
+* **Ready for review** - Issues for which there is at least one open Pull Request that has reviewers assigned to it, but which hasn't been reviewed yet.
+* **Reviewed with actions** - Issues for which there is at least one open Pull Request that has been reviewed and changes have been requested from the reviewer, but not yet actioned.
+* **Discuss** - Issues that should be discussed at an upcoming development meeting.
+* **Done** - Issues that have been addressed or are no longer valid.
+
+Notice that all the categories refer to [Issues](#issues).
+[Pull Requests](#pull-requests) should not be put on the project board.
```

Comment:
```suggestion
[Pull Requests](#pull-requests) should not be put on the project board, except if they do not have a linked issue. (See  [below](#issues) for special cases in which this might occur.)
```